packages/client/src/components/customers/CustomerForm/CustomCheckbox.tsx

```diff
@@ -9,16 +9,23 @@ import makeStyles from "@mui/styles/makeStyles";
 interface Props {
   name: string;
   label: string;
+  value?: string;
+  disabled?: boolean;
 }
 
-const CustomCheckbox: React.FC<Props> = ({ name, label }) => {
+const CustomCheckbox: React.FC<Props> = ({
+  name,
+  label,
+  disabled = false,
+  value,
+}) => {
   const classes = useStyles();
-  const [field] = useField({ name, type: "checkbox" });
+  const [field] = useField({ name, type: "checkbox", value });
```

Comment:
Is the value necessary here...my understanding is that it is stored in formik's state and retrieved + updated through `field` object.

Reply:
I see...(checkboxes and radios always eluded me). Disregard this ☝️